src/Menu/MenuList.js

```diff
@@ -18,7 +18,7 @@ function MenuList({children}) {
 			ref={popover.setRef}
 			arrow={popover.arrow}
 		>
-			<MenuListUI.Wrapper {...menuListProps}>
+			<MenuListUI.Wrapper role="menu" {...menuListProps}>
```

Comment:
This isn't necessary here, `menuListProps` already contains the `role`:
https://github.com/5app/base5-ui/blob/8d5949a660d48858f31d8d54f348b38617d48fb5/src/Menu/Menu.js#L114-L116

But it would be great if you could add `as="div"` here, and revert the changes made to the `styled.x` definitions made in [MenuList/index.js](https://github.com/5app/base5-ui/pull/199/files#diff-1b3f98f48f52b995a3179c2f672290a12ce03f14ef0e72d7d2f5834fac7dec9f). Those should still default to `ul`/`li` as before (for example for the `NavMenu` component which doesn't add any roles), and should only be overridden when a role is provided.

```suggestion
			<MenuListUI.Wrapper as="div" {...menuListProps}>
```

The same would have to be done for the `MenuItem` component: https://github.com/5app/base5-ui/blob/master/src/Menu/MenuItem.js#L22

Reply:
Ah okay, sorry, I completely misunderstood earlier. 😬